home.html

```diff
@@ -0,0 +1,90 @@
+</head>
+<body>
+    <header>
+            <a class="main_page" href="/"><img class="logo" src="logo.png" alt="판다마켓 로고"></a>
+            <a class="login_button" href="/login">로그인</a>
+    </header>
+
+    <section class="top">
+        <div class="container">
+            <div class="content">
+                    <h1 class="content-title">
+                        일상의 모든 물건을<br>
+                        거래해 보세요
+                    </h1>
+                    <a class="shopping-button" href="/items">구경하러 가기</a>
+            </div>
+            <img src="Img_home_top.png">
+        </div>
+    </section>
+
+    <main>
+        <section>
+            <div class="container">
+                <img src="Img_home_01.png">
+                <div class="content">
+                    <p class="badge">Hot item</p>
+                    <h1 class="content-title">인기 상품을<br>확인해보세요</h1>
+                    <p class="description">가장 HOT한 중고거래 물품을<br>판다 마켓에서 확인해 보세요</p>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div class="container">
+                <div class="content right">
+                    <p class="badge">Search</p>
+                    <h1 class="content-title">구매를 원하는<br>상품을 검색하세요</h1>
+                    <p class="description">구매하고 싶은 물품을 검색해서<br>쉽게 찾아보세요</p>
+                </div>
+                <img src="Img_home_02.png">
+            </div>
+        </section>
+        <section>
+            <div class="container">
+                <img src="Img_home_03.png">
+                <div class="content">
+                    <p class="badge">Register</p>
+                    <h1 class="content-title">판매를 원하는<br>상품을 등록하세요</h1>
+                    <p class="description">어떤 물건이든 판매하고 싶은 상품을<br>쉽게 등록하세요</p>
+                </div>
+            </div>
+        </section>
+    </main>
+
+    <section class="bottom">
+        <div class="container">
+            <div class="content">
+                <h1 class="content-title">
+                    믿을 수 있는<br>
+                    판다마켓 중고 거래
+                </h1>
+            </div>
+            <img src="Img_home_bottom.png">
+        </div>
+    </section>
+
+    <footer>
+        <div class="footer-wrap container">
+            <span class="codeit">@codeit - 2024</span>
+            <div class="link">
+                <a href="/privacy">Privacy Policy</a>
+                <a href="/faq">FAQ</a>
+            </div>
+            <div class="sns">
+                <a href="https://www.facebook.com/?locale=ko_KR" target="_blank"><img src="ic_facebook.png" alt="facebook"></a>
+                <a href="https://x.com/?lang=ko" target="_blank"><img src="ic_twitter.png" alt="twitter"></a>
+                <a href="https://www.youtube.com/" target="_blank"><img src="ic_youtube.png" alt="youtube"></a>
+                <a href="https://www.instagram.com/" target="_blank"><img src="ic_instagram.png" alt="instagram"></a>
+            </div>
```

Comment:
💊 제안
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
            <ul class="sns">
                <li>
                    <a href="https://www.facebook.com/?locale=ko_KR" target="_blank"><img src="ic_facebook.png" alt="facebook"></a>
                </li>
            </ul>
```